templates/authentication-required

```diff
@@ -38,20 +38,31 @@
   {{end}}
   {{ if .UseEmail }}
           <div>
-            <form method="post" action="{{ .WithEmailURL }}">
-              <label for="email">Email address</label>
-              <input type="email" id="email" name="email">
-              <button type="submit">Continue</button>
+            <form method="post" action="{{ .WithEmailURL }}" style="margin-bottom: 1rem;">
+              <div class="p-form-validation {{ if .Error  }}is-error{{ end }}">
+                <label for="email">Email address</label>
+                <div class="p-form--inline" style="width: 100%;">
+                  <div style="width: 100%;">
+                    <input type="email" id="email" name="email" class="p-form-validation__input">
```

Comment:
add `required` to prevent posting an empty string.

Reply:
done